### PR TITLE
JBPM-8770 fix for UnsupportedOperationException modifying an inmutable list.

### PR DESCRIPTION
**Pull request by gorlok**
Bug report on https://issues.jboss.org/browse/JBPM-8770

### CODE REVIEW

File: jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity/JBossUserGroupCallbackImpl.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public boolean existsGroup(String groupId) {
 		if( groups == null ) { 
 		    groups = new ArrayList<String>(0);
 		}
-		return groups;
+		return new java.util.ArrayList<String>(groups);
```
Comment (mswiderski) [inline jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity/JBossUserGroupCallbackImpl.java]: Maybe remove the package name as it is already in the import statements.

Comment (gorlok) [reply jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity/JBossUserGroupCallbackImpl.java]: Done. Thanks!

**Comment by mswiderski** [comment]
Ok to test

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
@gorlok thanks for contribution

**Pull request merged**